python/ray/ml/integrations/torch/torch_utils.py

```diff
@@ -0,0 +1,84 @@
+from typing import Optional, Union, List
+
+import pandas as pd
+import torch
+
+
```

Comment:
I'd suggest add unit test for a new lib/utility.

Reply:
Added tests!